SimPEG/DCIP/DCIPUtils.py

```diff
@@ -0,0 +1,1124 @@
+from __future__ import absolute_import
```

Comment:
@sgkang @micmitch @fourndo This is in the old `DCIP` folder. Is that supposed to be there or was this an accidental commit?